day_6/exercises/burrito.js

```diff
@@ -15,13 +15,31 @@ class Burrito {
     this.protein = protein;
     this.base = base;
     this.toppings = toppings;
-  }
+  };
+  addTopping(topping) {
+    this.toppings.push(topping);
+  };
+  removeTopping() {
+    this.toppings.pop();
+  };
+  changeProtein(protein) {
+    this.protein = protein;
+  };
```

Comment:
Good work defining your methods and updating the Burrito with `this`. A challenge would be to add an argument to `removeTopping(topping)` to delete a specific topping.